frontend/src/components/place/PlaceVisitedComponent.vue

```diff
@@ -0,0 +1,105 @@
+<template>
```

Comment:
Name PlaceVisitedComponent.vue is weird. We know that it is component already. PlaceVisitedPreview is enough